briana-olathe-ks.md

```diff
@@ -0,0 +1,10 @@
+# Olathe, KS 
+
+## Restaurants 
+- Joe's KC BBQ 
+- Kin Lin
+
```

Comment:
Any restaurants with a good selection of 🍺 

I'm also a big fan of 🌯 s, any places to go?